docs/modules/ROOT/pages/access.adoc

```diff
@@ -106,6 +106,22 @@ after an initial stage with centralized administration is over.
 WARNING: Removing the owner altogether will mean that administrative tasks that are protected by `assert_only_owner`
 will no longer be callable!
 
+=== Two step transfer
+
```

Comment:
i think we should also show the interface, as we do for the 1step impl

Reply:
![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExbWxsNmh2amo2Z2wxenowcjExNnIzdWlteDZoNjJsM290M3MyanVzMiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/pdSncNyYgaH0wqaCqp/giphy.gif)